specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -134,6 +134,21 @@ export function Notifications({
           count: notifications.length,
         })
       : notificationsText.notificationsLoading();
+
+  const onClearAll = () => {
```

Comment:
can you call this `handleClearAll` instead?
again consistency...

on*** when something emits events
handle*** when something handles an emitted event
quite common React convention